projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -433,14 +433,14 @@ private function parse_menu_item( $title ) {
 		if ( false !== strpos( $title, 'count-' ) ) {
 			preg_match( '/<span class=".+\s?count-(\d*).+\s?<\/span><\/span>/', $title, $matches );
 
-			$count = absint( $matches[1] );
+			$count = absint( $matches[1] ? $matches[1] : 0 );
 			if ( $count > 0 ) {
 				// Keep the counter in the item array.
 				$item['count'] = $count;
 			}
 
 			// Finally remove the markup.
-			$title = trim( str_replace( $matches[0], '', $title ) );
+			$title = trim( $matches[0] ? str_replace( $matches[0], '', $title ) : '' );
```

Comment:
Same here.
If you add the `preg_match()` into the `if` condition, this code will never run unless there's a match, so `$matches[0]` cannot be empty.